specs/template.md

```diff
@@ -49,15 +49,21 @@
 
 *Consider high and low-level APIs. Attempt to design a powerful and extensible low-level API with a high-level API for developer/designer ergonomics and simplicity.*
 
-### Anatomy and Appearance
+### Anatomy 
 
-*Screenshots and/or description of the basic appearance of the component. Outline its structure with a diagram of its visual tree (shadow dom). Enumerate key areas of visual customization, such as:*
+Outline the component structure with a diagram of its visual tree (shadow dom). Enumerate key areas of visual customization, such as:*
 
 - *Slot Names*
 - *Host Classes*
 - *Slotted Content/Slotted Classes*
 - *CSS Parts*
 
+*Work closely with the visual design partner to co-develop the API and anatomy along side the visual design.
+
+### Visual Appearance
+
+*Provide Figma files and other design assets. Be sure to account for the various component states, including hover, active, etc. as well as validity, and appearance variants. Consider all officially supported design systems as well as known community scenarios as appropriate. Work closely with engineering to co-develop the visual design along side the API and anatomy.*
```

Comment:
I may want to tweak this a bit for nuance. Technically the only officially supported design system from a visual appearance standpoint is our own. I think perhaps the intent here is to ensure that our template and class are flexible to accommodate other design systems (which may be included as "prior art" or industry examples above), but I don't think we should try and design for systems the community has *extended* off FAST. I'm not sure there's a way for us to accommodate that.

Reply:
I'll take a stab at the re-wording.

```suggestion
*Provide Figma files and other design assets. Be sure to account for the various component states, including hover, active, etc. as well as validity, and appearance variants. Focus primarily on the officially supported design system as well as known community scenarios as appropriate. Consider other popular design systems during this process and, where possible, ensure that common design features that may not be part of the officially supported design system can be accommodated. Work closely with engineering to co-develop the visual design along side the API and anatomy.*
```